frontend/packages/dev-console/src/components/pipelines/pipeline-topology/BuilderNode.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { PlusIcon } from '@patternfly/react-icons';
+import { Node, observer } from '@console/topology';
+import { AddNodeDirection, BUILDER_NODE_ADD_RADIUS } from './const';
+import TaskNode from './TaskNode';
+import { BuilderNodeModelData } from './types';
+
+import './BuilderNode.scss';
+
+const drawAdd = (x, y, onClick) => {
+  return (
+    <g className="odc-builder-node__add-icon" onClick={onClick} transform={`translate(${x}, ${y})`}>
+      <circle cx={0} cy={0} r={BUILDER_NODE_ADD_RADIUS} fill="blue" />
+      <g transform="translate(-6, -6)">
+        <PlusIcon color="white" />
+      </g>
+    </g>
+  );
+};
+
+const BuilderNode: React.FC<{ element: Node }> = ({ element }) => {
+  const [showAdd, setShowAdd] = React.useState(false);
+  const { width, height } = element.getBounds();
+  const { onAddNode } = element.getData() as BuilderNodeModelData;
+
+  return (
+    <g
+      className="odc-builder-node"
+      onFocus={() => setShowAdd(true)}
+      onBlur={() => setShowAdd(false)}
+      onMouseOver={() => setShowAdd(true)}
+      onMouseOut={() => setShowAdd(false)}
```

Comment:
You could use the topology package `useHover` here for mouse over and mouse out. That gives you the default delay for changing state on hovers.

Reply:
Oooh, how fancy. I'll be sure to update it to do that.